boston_logger/sensitive_paths.py

```diff
@@ -184,3 +185,37 @@ def sanitize_data(data: dict, *mask_names):
         _mask_processors[mask_name].process(masked_data)
 
     return masked_data
+
+
+def sanitize_url(url: str, *mask_names):
+    """Return copy of url with query string that has been sanitized.
+
+    Global masks, the current SensitivePathContext and any positional args will
+    be applied.
+    """
+
+    try:
+        parts = urlparse(url)
+        return urlunparse(parts._replace(query=sanitize_string(parts.query)))
+    except ValueError:
+        from .config import config
+
+        return MASK_STRING if config.PREFER_TEXT_FALLBACK_MASKING else url
+
+
+def sanitize_string(data: str, *mask_names):
```

Comment:
This may be a little nitpicky, but I'd consider calling this sanitize_querystring instead of naming it after a generic string.

Reply:
That's probably a good idea.